README.md

```diff
@@ -118,19 +118,21 @@ viewer will be able to highlight text found via a search, and display a search i
 within the viewer.
 
 ### Setting up hOCR
+
 To display a text overlay, Mirador must be provided with hOCR text data - which is OCR'd text that includes position information for the extracted text relative to the image that is being displayed. Here are the steps:
-1. Go to "Administration » Structure » Media Types", select the "**File**" media type, and click "**Manage Fields**".
-2. Add a new field to the **File** media type called "**hOCR extracted Text**". Set the allowed file extensions to "xml"<br />![media-file-field_hocr_extracted-file-label.png](docs%2Fmedia-file-field_hocr_extracted-file-label.png)  ![media-file-field_hocr_extracted-file-extensions.png](docs%2Fmedia-file-field_hocr_extracted-file-extensions.png)
-3. Go to "Administration » Configuration » System » Actions" and click "**Create New Advanced Action**" with the "**Generate Extracted Text for Media Attachment**" action type.<br />![action-hocr-extracted-text.png](docs%2Faction-hocr-extracted-text.png)<br />
-![action-hocr-extracted-text-config.png](docs%2Faction-hocr-extracted-text-config.png)<br />
-    - Give the new action a name that mentions hOCR.<br />
-    - In Format field select hOCR Extracted Text with Positional Data
-    - For Destination File Field Name select the field you just created (`field_hocr_extracted_text`)
-    - Keep *None* for the destination text field
-    - And save the action
-4. Go to " Administration » Structure » Context" and edit the **Page Derivatives** context<br />![context-paged-derivatives-add-reaction.png](docs%2Fcontext-paged-derivatives-add-reaction.png)
-    - Click **Add Reaction** and choose "**Derive File for Existing Media**"
-    - In the select box choose the action you created above and save.
+
+1. Ensure you're running isle-buildkit version 3.2.6 or above
+2. Install the Drupal modules https://github.com/discoverygarden/islandora_hocr and https://github.com/Born-Digital-US/islandora_iiif_hocr
+3. Add
+```
+  <searchComponent
+      class="solrocr.OcrHighlightComponent"
```

Comment:
This is not necessary if you set a variable in your docker-compose.yml:

In build/docker-compose/docker-compose.drupal.yml in the environment section, add

SOLR_HOCR_PLUGIN_PATH: ${SOLR_HOCR_PLUGIN_PATH}

Then when make solr-cores runs, the necessary config settings for hOCR will be generated by islandora_hocr.

see it in this branch: 
https://github.com/Islandora-Devops/isle-dc/blob/solr-hocr/build/docker-compose/docker-compose.drupal.yml

Reply:
I don't see `SOLR_HOCR_PLUGIN_PATH` defined anywhere in [the Solr OCR Highlighting plugins' install docs](https://dbmdz.github.io/solr-ocrhighlighting/0.9.0/installation/). IIUC if the plugin is placed in a common solr lib it will automatically get loaded.

That being said, I have not tested a solr instance without this setting - I added it per their install instructions. I'll see if this is maybe not needed with the plugin getting loaded.

Reply:
It's read by islandora_hocr, if it sees that environment variable, it will add all of the configs needed to load the ocr highlighting library when you download the Solr configs from Drupal.

Reply:
Thanks @alxp - I updated the docs to reflect this. Once https://github.com/Islandora-Devops/isle-buildkit/pull/345 I think we can merge this PR